### ISSUE TITLE
'profile' scope causes Federated Identity example to loop

### ISSUE THREAD

**Issue by sjerman** [spring-projects/spring-authorization-server#919]
**Expected Behavior**
Allow addition of 'profile' scope.

**Current Behavior**
If you add the scope to the example, starts infinite redirect loop.

**Context**
I'm trying to make use of the Federated Identity example and would I'd like to get the UserInfo object populated. However if I add the profile to the registeredClient I get the hang:

In AuthorizationServerConfig:

```
   RegisteredClient registeredClient =
        RegisteredClient.withId(UUID.randomUUID().toString())
            .clientId("messaging-client")
            .clientSecret("{noop}secret")
            .clientAuthenticationMethod(ClientAuthenticationMethod.CLIENT_SECRET_BASIC)
            .authorizationGrantType(AuthorizationGrantType.AUTHORIZATION_CODE)
            .authorizationGrantType(AuthorizationGrantType.REFRESH_TOKEN)
            .authorizationGrantType(AuthorizationGrantType.CLIENT_CREDENTIALS)
            .redirectUri("http://127.0.0.1:8080/login/oauth2/code/messaging-client-oidc")
            .redirectUri("http://127.0.0.1:8080/authorized")
            .scope(OidcScopes.OPENID)
            .scope("message.read")
            .scope("message.write")
            // .scope(OidcScopes.PROFILE)
            .clientSettings(ClientSettings.builder().requireAuthorizationConsent(false).build())
            .build();
```

**Comment by sjerman** [comment]
fixed via comment on #917. Issue is that sample doesn't;t include 

```
       .oauth2ResourceServer(OAuth2ResourceServerConfigurer::jwt);
```

and

```
  @Bean
  public JwtDecoder jwtDecoder(JWKSource<SecurityContext> jwkSource) {
    return OAuth2AuthorizationServerConfiguration.jwtDecoder(jwkSource);
  }
```

**Comment by jgrandja** [comment]
Duplicate of gh-917
